src/Symfony/Component/Console/Command/SignalableCommandInterface.php

```diff
@@ -20,8 +23,11 @@ interface SignalableCommandInterface
 {
     /**
      * Returns the list of signals to subscribe.
+     *
+     * @param InputInterface  $input
+     * @param OutputInterface $output
      */
-    public function getSubscribedSignals(): array;
+    public function getSubscribedSignals(/* InputInterface $input, OutputInterface $output */): array;
```

Comment:
you cannot add mandatory arguments in a method signature through such BC layer, because the child class is not allowed to add them as mandatory arguments.

Btw, changing the subscribed signals based on the *output* looks weird to me. What is the use case for it ? Is this worth a BC break ?

Reply:
The more I thought about it, the less sense it made to me as well. I initially had a use case for #53508, but I've since found a better approach, and now I can't really think of a good use case. Therefore, I've removed the entire commit. If an actual use case comes up in the future, it can be added then. I've also removed the `$output` argument from `getAlarmTime()` as it was also useless.